api/src/main/java/org/pmiops/workbench/cohortbuilder/ParticipantCounter.java

```diff
@@ -142,8 +141,8 @@ public QueryJobConfiguration buildQuery(ParticipantCriteria participantCriteria,
      * @param hi - upper bound of the age range
      * @return
      */
-    private static String getAgeRangeSql(int lo, int hi) {
-        return "when CAST(FLOOR(DATE_DIFF(CURRENT_DATE, DATE(person.year_of_birth, person.month_of_birth, person.day_of_birth), MONTH)/12) as INT64) >= " + lo +
-                " and CAST(FLOOR(DATE_DIFF(CURRENT_DATE, DATE(person.year_of_birth, person.month_of_birth, person.day_of_birth), MONTH)/12) as INT64) <= " + hi + " then '" + lo + "-" + hi + "'";
+    private static String getAgeRangeSql(int lo, int hi, String table) {
+      return "when CAST(FLOOR(DATE_DIFF(CURRENT_DATE, DATE(" + table + ".dob), MONTH)/12) as INT64) >= " + lo +
```

Comment:
I don't think the parameterization makes sense here, since the original "person" table doesn't have a column called "dob" - this only works on search_person.